### PR TITLE
Mrc 130 amount above or below data entry limits

### PR DESCRIPTION
**Pull request by PavelVjalicin**
Refactored positiveWholeAmountFormatter to wholeAmountFormatter. Removed some tests. WIll need to add additional tests.

### CODE REVIEW

File: app/forms/DistributionsIncludedFormProvider.scala
Comment (girishkamat) [inline app/forms/DistributionsIncludedFormProvider.scala:39]: Can we move this to a constant please - 1000000000

File: conf/messages.en
Comment (girishkamat) [inline conf/messages.en:41]: This would mean 1000000000 is not allowed. Actually anything greater than 1000000000 is not allowed

File: app/forms/DistributionsIncludedFormProvider.scala
Comment (PavelVjalicin) [reply app/forms/DistributionsIncludedFormProvider.scala:39]: Where would the constant be located? In utils?
Something like we had before?

val ONE_BILLION = 1000000000

And I assume that we would then use the constant everywhere we encounter "1000000000" value?

File: app/forms/mappings/Formatters.scala
Comment (girishkamat) [inline app/forms/mappings/Formatters.scala:129]: Do we need the range implementation here?

File: app/forms/mappings/Formatters.scala
Comment (girishkamat) [inline app/forms/mappings/Formatters.scala:126]: Why do we need this ```matches(WholeNumber)```?

File: app/forms/mappings/Formatters.scala
Comment (PavelVjalicin) [reply app/forms/mappings/Formatters.scala:129]: I added it as a worst-case scenario fallback. Just in case. Was not confortable with throwing a number format exception. Ideally, this should not be triggered ever because of constraints.

File: app/forms/mappings/Formatters.scala
Comment (PavelVjalicin) [reply app/forms/mappings/Formatters.scala:126]: To distinguish between nonNumericKey error and outOfRangeKey error. Regex confirms that it's an actual numeric value.

File: conf/messages.en
Comment (PavelVjalicin) [reply conf/messages.en:41]: Yes. I would suggest something like "Enter an amount lower than or equal to 1000000000" Who should I confirm this with? Darryl?

File: app/forms/mappings/Formatters.scala
Comment (PavelVjalicin) [reply app/forms/mappings/Formatters.scala:126]: If no match is present for example "100000000000000000000000000" returns nonNumericKey error.

File: conf/messages.en
Comment (girishkamat) [reply conf/messages.en:41]: Yes, with Darryl

**Comment by PavelVjalicin** [comment]
Should be good to go now. Blocked by wording for greater than one billion error message. Will merge after approval.

**Review by girishkamat — APPROVED** [review]

**Pull request merged**
